### PR TITLE
Specify Trivy version in workflow file

### PR DESCRIPTION
**Pull request by oyri**
To enable to use previous version av Trivy when breaking changes occur before container-scan action is updated.
If not set, default is latest version.

### CODE REVIEW

File: src/trivyHelper.ts [hunk 1]
```diff
@@ -57,8 +59,12 @@ export async function runTrivy(): Promise<TrivyResult> {
 }
 
 export async function getTrivy(): Promise<string> {
-    const latestTrivyVersion = await getLatestTrivyVersion();
 
+    let latestTrivyVersion = inputHelper.trivyVersion;
+    if(trivyVersion == 'latest'){
+        latestTrivyVersion = await getLatestTrivyVersion();
+    }
```
Comment (ckotzbauer) [inline src/trivyHelper.ts:66]: I think the imports are mixed up here (`inputHelper.trivyVersion` and `trivyVersion` are the same from different imports).
Suggestion:
```
let latestTrivyVersion = inputHelper.trivyVersion;
if (latestTrivyVersion === 'latest') {
  latestTrivyVersion = await getLatestTrivyVersion();
}
```

**Review by ckotzbauer — COMMENTED** [review]
Thanks for this PR 🎉

File: src/trivyHelper.ts (hunk 1 above)
Comment (oyri) [reply src/trivyHelper.ts:66]: Yes, thank you for that! I overlooked that one.

**Comment by Viktork25** [comment]
Can I suggest that trying to use a different version we will need to do changes also in the way that trivy scans the image?Because in the latest version 0.23 of trivy, the trivy run with the following structure `trivy image [command options] image_name` and on old version runs like this `trivy [image-name]`

**Comment by oyri** [comment]
> Can I suggest that trying to use a different version we will need to do changes also in the way that trivy scans the image?Because in the latest version 0.23 of trivy, the trivy run with the following structure `trivy image [command options] image_name` and on old version runs like this `trivy [image-name]`

I didn't add this since it's already fixed in a different pull request, so I didn't want to cause unessasary merge conflicts... Is it not better to include pull request https://github.com/Azure/container-scan/pull/123 first before mine?

**Review by scottwestover — COMMENTED** [review]
Thanks for opening the PR! 
For the new input, it might be worth adding to the readme here: https://github.com/Azure/container-scan#action-inputs

**Comment by oyri** [comment]
> Thanks for opening the PR! For the new input, it might be worth adding to the readme here: https://github.com/Azure/container-scan#action-inputs

I thought the release-branch was the correct to merge to and this one does not have a readme with this info, so maybe I should rather merge to master and then update the readme as well. Good point to update the readme.

**Comment by oyri** [comment]
Replaced by #130, close this one.

**Pull request closed without merging**